ecal/core/src/ecal_registration_provider.cpp

```diff
@@ -320,7 +321,18 @@ namespace eCAL
     std::lock_guard<std::mutex> lock(m_server_map_sync);
     for(SampleMapT::const_iterator iter = m_server_map.begin(); iter != m_server_map.end(); ++iter)
     {
-      // register sample
+      //////////////////////////////////////////////
+      // update description
+      //////////////////////////////////////////////
+      auto& ecal_sample_service = iter->second.service();
```

Comment:
I guess this can be made const? please double check

```suggestion
      const auto& ecal_sample_service = iter->second.service();
```